assets/js/nebula.js

```diff
@@ -2728,23 +2728,34 @@ nebula.cf7LocalStorage = function(){
 		return false;
 	}
 
-	jQuery('.wpcf7-textarea, .wpcf7-text').each(function(){
+	jQuery('.wpcf7-textarea, .wpcf7-text, .wpcf7-select, .wpcf7-checkbox input').each(function(){
 		var thisLocalStorageVal = localStorage.getItem('cf7_' + jQuery(this).attr('name'));
 
-		//Fill textareas with localstorage data on load
+		//Complete inputs with localstorage data on load
 		if ( !jQuery(this).hasClass('do-not-store') && !jQuery(this).hasClass('.wpcf7-captchar') && thisLocalStorageVal && thisLocalStorageVal !== 'undefined' && thisLocalStorageVal !== '' ){
-			if ( jQuery(this).val() === '' ){ //Don't overwrite a field that already has text in it!
+			if( jQuery(this).attr('type') == 'checkbox' ){
```

Comment:
Minor consistency for Nebula

```suggestion
			if ( jQuery(this).attr('type') == 'checkbox' ){
```